src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Controllers/DataSetsControllerTests.cs

```diff
@@ -714,440 +719,859 @@ public GetDataSetMetaTests(TestApplicationFactory testApp) : base(testApp)
         {
         }
 
-        [Fact]
-        public async Task VersionExists_ReturnsCorrectViewModel()
+        public class NoQueryParametersTests : GetDataSetMetaTests
         {
-            DataSet dataSet = DataFixture
-                .DefaultDataSet()
-                .WithStatusPublished();
-
-            await TestApp.AddTestData<PublicDataDbContext>(context => context.DataSets.Add(dataSet));
-
-            var filterMetas = DataFixture
-                .DefaultFilterMeta()
-                .WithOptions(() => DataFixture
-                    .DefaultFilterOptionMeta()
-                    .GenerateList(3))
-                .GenerateList(3);
+            public NoQueryParametersTests(TestApplicationFactory testApp) : base(testApp)
```

Comment:
We're getting a few primary constructor suggestions (in Rider) now. As we discussed, it's okay to start using these now so we might as well do it for the various nested classes in this test suite